src/tree/utils/populateTree.ts

```diff
@@ -192,6 +192,25 @@ function processObject(
   return node;
 }
 
+function processRef(
+  node: TreeListNode,
+  schema: JSONSchema4,
+  level: number,
+  path: JsonPath,
+  options: WalkingOptions | null,
+) {
+  (node as TreeListParentNode).children = [];
+  try {
+    const resolved = resolveSchema(schema, path, options);
```

Comment:
I suppose that won't have any performance impact? It's being resolved elsewhere anyway if I'm correct.

Reply:
Yeah, it shouldn't really have any.